tools/scripts/update_sorbet_ruby_patch.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
TIL that there exists this magic `.patch` URL thing on GitHub that would *almost* work as a stand-in for this script, e.g.:

https://github.com/sorbet/ruby/compare/v2_7_2...sorbet_ruby_2_7.patch

But unfortunately this includes the `a/` and `b/` paths that we need to avoid by passing `--no-prefix`, so I don't think we can use it directly here.

Reply:
You could use the `patch_args` argument to `http_archive` to pass in `-p 1`.

Reply:
Ooh, didn't realize `patch_args` was a thing. I'll give it a shot.

Reply:
Seems to work! I went ahead and pushed a change to the script.

We could probably actually make use of `remote_patch` to avoid the need to store the patch in the repo entirely. (Might still want a bump script, but it would just need to update a git SHA and a checksum.) What do you think of that option?

Reply:
Ah, never mind --- apparently that's not available in 4.1.0.